web_quick_start_screen/models/quick_start_screen_action.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class DecontracStartScreenAction(models.Model):
+    _name = "quick.start.screen.action"
+    _description = "Configure actions for for a given model using the mixin"
+    _order = "sequence, id"
+
+    active = fields.Boolean(default=True)
+    sequence = fields.Integer()
+    name = fields.Char()
+    image = fields.Image("Start screen icon", max_width=128, max_height=128, store=True)
+    icon_name = fields.Char(help="Just set the Font Awesome icon name. e.g.: fa-truck")
+    color = fields.Integer(help="Choose the icon color")
+    description = fields.Html()
+    action_ref_id = fields.Reference(
+        selection=[
+            ("ir.actions.report", "ir.actions.report"),
```

Comment:
Should we put nicer labels?

Reply:
In this case, I think it's better to reflect for sure what model we're taking the record from